src/main/charts/confluence/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: confluence
 description: A chart for installing Confluence Data Center on Kubernetes
 type: application
-version: 0.14.0
+version: 0.15.0
 appVersion: 7.12.3-jdk11
```

Comment:
`7.12.3-jdk11` is the latest released version, we can update it
cc @badgersow

Reply:
`7.12.4-jdk11` is the latest version

Reply:
🤦 thinking one number, writing different number :) Yes, I've meant 7.12.4